src/test/java/org/prebid/server/bidder/adquery/AdQueryBidderTest.java

```diff
@@ -0,0 +1,480 @@
+                        .requestId("abs")
+                        .cpm("123")
+                        .adqLib("AnyLib")
+                        .tag("AnyTag")
+                        .adDomains(singletonList("any"))
+                        .creationId(312)
+                        .adQueryMediaType(AdQueryMediaType.of(BidType.banner, "120", "320"))
+                        .build())));
+
+        // when
+        final Result<List<BidderBid>> result = target.makeBids(httpCall, BidRequest.builder().id("abs1").build());
+
+        // then
+        assertThat(result.getValue())
+                .extracting(BidderBid::getBidCurrency)
+                .hasSize(1)
+                .containsExactly("PLN");
+        assertThat(result.getValue())
+                .extracting(BidderBid::getType)
+                .hasSize(1)
+                .containsExactly(BidType.banner);
+        assertThat(result.getValue())
+                .hasSize(1)
+                .extracting(BidderBid::getBid)
+                .containsExactly(Bid.builder()
+                        .id("abs")
+                        .impid("1")
+                        .price(BigDecimal.valueOf(123))
+                        .adm("<script src=\"AnyLib\"></script>AnyTag")
+                        .adomain(singletonList("any"))
+                        .crid("312")
+                        .w(120)
+                        .h(320)
+                        .build());
+        assertThat(result.getErrors()).isEmpty();
+    }
+
+    @Test
+    public void makeBidsShouldReturnThrowErrorWheResponseBodyAdQueryMediaTypeWidthIsInvalid()
+            throws JsonProcessingException {
+        // given
+        final BidderCall<AdQueryRequest> httpCall = givenHttpCall(null,
+                mapper.writeValueAsString(AdQueryResponse.of(AdQueryDataResponse.builder()
+                        .requestId("abs")
+                        .cpm("123")
+                        .adqLib("AnyLib")
+                        .tag("AnyTag")
+                        .adDomains(singletonList("any"))
+                        .creationId(312)
+                        .adQueryMediaType(AdQueryMediaType.of(BidType.banner, "ab", "320"))
+                        .build())));
+
+        // when
+        final Result<List<BidderBid>> result = target.makeBids(httpCall, BidRequest.builder().id("abs1").build());
+
+        // then
+        assertThat(result.getValue()).isEmpty();
+        assertThat(result.getErrors()).hasSize(1)
+                .containsExactly(BidderError.badServerResponse("Value of measure: ab can not be parsed."));
+    }
+
+    @Test
+    public void makeBidsShouldReturnThrowErrorWheResponseBodyAdQueryMediaTypeHeightIsInvalid()
+            throws JsonProcessingException {
+        // given
+        final BidderCall<AdQueryRequest> httpCall = givenHttpCall(null,
+                mapper.writeValueAsString(AdQueryResponse.of(AdQueryDataResponse.builder()
```

Comment:
mapping and creating AdQueryResponse repeat in every test, maybe it's better to extract inside the givenHttpCall method